bots/MyUtility.lua

```diff
@@ -405,17 +405,58 @@ function U.GetWeakestUnit(units)
 	return lowestUnit;
 end
 
-function GetItemWardSlot(bot)
-	return bot:FindItemSlot("item_ward_observer")
+local function GetItem(bot, item_name)
+   local itemSlot = bot:FindItemSlot(item_name)
+   if itemSlot == -1 then
```

Comment:
Can you add the check to know if the slot the item is in is greater than 8? Reason being that, it means the item is in the backpack/stash/courier and the bot won't be able to use the item without adding more logic to swap it in (not a prioroty now)